spec/index.bs

```diff
@@ -231,7 +231,7 @@ interface GPUTextureView : GPUObjectBase {
 };
 
 interface GPUTexture : GPUObjectBase {
-    GPUTextureView createView(GPUTextureViewDescriptor desc);
```

Comment:
Descriptors are everywhere here so I think abbreviating to `desc` is reasonable.

Reply:
I think this was for consistency with all the `GPUDevice` methods that take descriptors and call them `descriptor`.

Reply:
Yup. Just for consistency, I don't actually care about the name.